twake/frontend/src/app/services/Globals.js

```diff
@@ -203,8 +224,8 @@ class Globals {
   }
 
   localStorageGetItem(key, callback) {
-      // eslint-disable-next-line no-undef
-      if (typeof SyncStorage != 'undefined') {
+    // eslint-disable-next-line no-undef
+    if (typeof SyncStorage != 'undefined') {
```

Comment:
`if (SyncStorage) {` is generally enough

Reply:
Not in the scope of your issue @stephanevieira75

Reply:
But thanks for the comment Christophe 👍